crates/zed/src/languages/astro/indents.scm

```diff
@@ -0,0 +1 @@
+; inherits: html
```

Comment:
Zed doesn't use this file. Can you remove?

Reply:
Sure! Sounds good.